prime-router/src/main/kotlin/cli/tests/CompareData.kt

```diff
@@ -0,0 +1,692 @@
+package gov.cdc.prime.router.cli.tests
```

Comment:
Shouldn't we move TestReportStream down into this tests package too?  Again, sounds like a future task

Reply:
I can move it.  Not a problem.